sklearn/gaussian_process/gpr.py

```diff
@@ -401,6 +404,10 @@ def log_marginal_likelihood(self, theta=None, eval_gradient=False):
             to the kernel hyperparameters at position theta is returned
             additionally. If True, theta must not be None.
 
+        clone_kernel : bool, default: True
```

Comment:
please add a test for this, (checking that `theta.kernel_.theta` it's changed upon a call to the `log_marginal_likelihood` with `clone_kernel=True`), but changes with `clone_kernel=False`.

Thanks!

Reply:
@c-bata any chance you might be able to add that test?

Reply:
Sorry for late reply. I added tests. Thanks!